packages/js-dpp/lib/stateTransition/AbstractStateTransitionIdentitySigned.js

```diff
@@ -121,6 +122,16 @@ class AbstractStateTransitionIdentitySigned extends AbstractStateTransition {
    * and purpose to sign this state transition
    */
   verifyPublicKeyLevelAndPurpose(publicKey) {
+    if (
```

Comment:
Shouldn't it be vice-versa? I mean it should go from what state transition require but not from what key we are passing. Isn't?